commit d8324a519ad419a335ab162010c1d7f8f36e35ae (refs/changes/31/4431/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-06-26T11:14:36+00:00 (1 year, 3 months ago)
    
    TRITON-549 Update javascriptlint and jsstyle in vmapi repo
    Reviewed by: Pedro P. Candel <pedro@joyent.com>
    Approved by: Pedro P. Candel <pedro@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index 7758103..4f2ec95 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -3,10 +3,10 @@
 	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/joyent/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
 	url = https://github.com/joyent/sdc-scripts.git
diff --git a/deps/jsstyle b/deps/jsstyle
index 9600c7e..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit 9600c7e56c84f3a74d6e3d70c336e86f7a3e3769
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
